src/components/research-form-wizard/FormPages/Motivation.tsx

```diff
@@ -1,5 +1,5 @@
 import { useResearchWizardStore } from '@/stores/researchWizard';
-import { useEffect, useState } from 'react';
+import { use, useEffect, useRef, useState } from 'react';
```

Comment:
Seems like we don't need `use here`:

```suggestion
import { useEffect, useRef, useState } from 'react';
```